packages/oasis-actions/src/operations/aave/openStEth.ts

```diff
@@ -80,9 +82,23 @@ export async function openStEth(
     to: addresses.operationExecutor,
   })
 
+  let borrower;
+
+  switch(args.flashloanType) {
```

Comment:
OK, I assume this PR is simply to demonstrate Balancer works. Then we can follow it up with Operation optimisations as a separate piece of work. Makes sense

Reply:
What do you mean by that?

Reply:
The point of Balancer is we can reduce the set of  actions and protocol touchpoints.
So instead we could have EG STETH:ETH position:
1. Flashloan WETH
2. Swap into STETH
3. Deposit STETH
4. Borrow WETH
Is handled automatically -> Repay WETH Flashloan with in-hand WETH and ETH deposited at start of t/x. We should always be able to borrow enough WETH post STETH deposit because our calculations will assume worse case scenario slippage on Swap. So, we'll be able to borrow what we need plus our initial deposit to repay FL at end.

VS Currently:
1. Flashloan DAI
2. Deposit DAI
3. Borrow WETH
4. Swap into STETH
5. Deposit STETH
6. Withdraw DAI
Repay DAI Flashloan